server/queue/queue-api/src/test/java/org/apache/james/queue/api/DelayedMailQueueContract.java

```diff
@@ -67,7 +67,7 @@ default void enqueueWithNegativeDelayShouldNotDelayDelivery(ExecutorService exec
     default void enqueueWithReasonablyLongDelayShouldDelayMail(ExecutorService executorService) throws Exception {
         getMailQueue().enQueue(defaultMail()
             .build(),
-            365 * 1000,
+            365 * 10,
```

Comment:
that's 10 years, not 100, no (see commit message)?

Reply:
yes, the commit describe the other change, but I can put 100 here too if you want (I think 10 years is reasonable long)

Reply:
you can keep it as such, I misread.